cwms-data-api/src/test/java/cwms/cda/api/TimeseriesControllerTestIT.java

```diff
@@ -1797,4 +1803,388 @@ enum GetAllTest
+                "/cwms/cda/api/spk/elev_ts_create.json");
+        assertNotNull(resource);
+        String tsData = IOUtils.toString(resource, StandardCharsets.UTF_8);
+
+        JsonNode ts = mapper.readTree(tsData);
+        String location = ts.get(NAME).asText().split("\\.")[0];
+        String officeId = ts.get("office-id").asText();
+
+        createLocation(location, true, officeId);  // This marks for delete at end of test.
+        updateLocation(location, true, officeId);
+
+        TestAccounts.KeyUser user = TestAccounts.KeyUser.SPK_NORMAL;
+
+        // inserting the time series
+        given()
+            .log().ifValidationFails(LogDetail.ALL, true)
+            .accept(Formats.JSONV2)
+            .contentType(Formats.JSONV2)
+            .body(tsData)
+            .header("Authorization",user.toHeaderValue())
+            .queryParam(OFFICE, officeId)
+        .when()
+            .redirects().follow(true)
+            .redirects().max(3)
+            .post("/timeseries/")
+        .then()
+            .log().ifValidationFails(LogDetail.ALL,true)
+        .assertThat()
+            .statusCode(is(HttpServletResponse.SC_OK));
+
+
+        System.out.println("Data has been inserted for " + location);
+
+        // 1209654000000 as ms == Thursday, May 1, 2008 3:00:00 PM
+
+        // get it back
+        String firstPoint = "2008-05-01T03:00:00.000Z";
+
+        // try once with auth
+        ValidatableResponse validatableResponse = given()
+                .log().ifValidationFails(LogDetail.ALL, true)
+                .header("Authorization",user.toHeaderValue())
+                .accept(Formats.JSONV2)
+                .queryParam(OFFICE, officeId)
+                .queryParam(UNIT, "m")
+                .queryParam(NAME, ts.get(NAME).asText())
+                .queryParam(BEGIN, firstPoint)
+            .when()
+                .redirects().follow(true)
+                .redirects().max(3)
+                .get("/timeseries/")
+            .then()
+                .log().ifValidationFails(LogDetail.ALL, true)
+            .assertThat()
+                .statusCode(is(HttpServletResponse.SC_OK));
+
+        System.out.println(validatableResponse.extract().asString());
+ // verify that there is vertical-datum-info in the response.
+        validatableResponse.body("vertical-datum-info", notNullValue())
+                .body("vertical-datum-info.location", equalTo(location))
+                .body("vertical-datum-info.unit", equalTo("m"))
+                .body("vertical-datum-info.offsets.size()", equalTo(1))
+                ;
+
+        // Try again without auth
+        validatableResponse = given()
+                .log().ifValidationFails(LogDetail.ALL, true)
+                .accept(Formats.JSONV2)
+                .queryParam(OFFICE, officeId)
+                .queryParam(UNIT, "m")
+                .queryParam(NAME, ts.get(NAME).asText())
+                .queryParam(BEGIN, firstPoint)
+                .when()
```

Comment:
Formatting for the when, then, and assertThat is incorrect, follow the format above